approval_polls/tests.py

```diff
@@ -8,14 +8,16 @@
 
 from approval_polls.models import Poll
 
-def create_poll(question, days=0, ballots=0):
+def create_poll(question, username, days=0, ballots=0):
```

Comment:
Can make username have a default value `username=user1` so you don't have to type it in repeatedly in later calls